### DISCUSSION TITLE
How to customize sass variable from file other than var.scss

### DISCUSSION THREAD

**Discussion by RizkiNfs** [element-plus/element-plus#12846] [Q&A] [answered]
Helo, I want to change variable from /theme-chalk/src/form.scss. I tried
```
@forward 'element-plus/theme-chalk/src/common/var.scss' with (
  $form-item-label-top-margin-bottom: (
    'large': 8px,
    'default': 6px,
    'small': 4px,
  )
);
```
It's doesn't work, I got `[sass] This variable was not declared with !default in the @used module.` error

**Comment by ryuhangyeong** [comment, marked as answer]
See element-plus-vite-starter.

- https://github.com/element-plus/element-plus-vite-starter
- https://github.com/element-plus/element-plus-vite-starter/blob/main/src/styles/element/index.scss
